front/components/editor/extensions/agent_builder/InstructionBlockExtension.tsx

```diff
@@ -305,7 +278,11 @@ export const InstructionBlockExtension =
               $from.parent.type.name === "paragraph" &&
               $from.parent.content.size === 0;
 
-            const content = buildInstructionBlockContent(type);
+            const content = {
```

Comment:
we are in tiptap so we can't rely on markdown, so we keep the JSON representation so it can be fed to tiptap